app/src/models/item.ts

```diff
@@ -8,90 +23,82 @@ import { Maniiifest } from "maniiifest";
 // TO DO: add isCartographic for map stuff
 export class ItemModel {
   uuid: string;
-  mods: any;
-  capture: any;
-  typeOfResource: string;
-  title: string;
-  isSingleCapture: boolean;
-  imageIDs?: string[];
-  href: string;
-  contentType: string;
-  hasItems?: boolean;
-  manifestURL: string;
   link: string;
+  manifestURL: string;
+  hasItems: boolean;
+  imageIDs: string[] | null;
+  metadata: NormalizedItemMetadata;
+  renderableMetadata: Record<string, string>;
+  title: string;
+  typeOfResource: string;
   isRestricted: boolean;
-  isImage: boolean;
-  archivesLink?: string | null | undefined;
-  catalogLink?: string | null | undefined;
   divisionLink: string;
-  metadata?: {
-    title: string;
-    names?: string;
-    collection?: string;
-    origin: string;
-    dateCreated?: string;
-    dateIssued: string;
-    tableOfContents?: string;
-    locations: string;
-    subjects?: string;
-    genres: string;
-    notes?: string;
-    physicalDescription?: string;
-    abstract?: string;
-    languages: string;
-    link?: string;
-    identifiers: string;
-    access: string;
-    rights: string;
-    typeOfResource: string;
-  };
+  contentType: string;
+  isImage: boolean;
+  archivesLink: string | null;
+  catalogLink: string | null;
+  citationData: CitationOutput;
+  breadcrumbData: any;
 
   constructor(uuid: string, manifest: any) {
-    // get custom label from manifest
     const parser = new Maniiifest(manifest);
-    this.hasItems = manifest.items.length > 0 ? true : false;
-    const canvases = Array.from(parser.iterateManifestCanvas());
-    const canvasAnnotations = Array.from(
-      parser.iterateManifestCanvasAnnotation()
-    );
-
-    // example canvas.id is: "https://iiif.nypl.org/iiif/3/TH-38454/full/!700,700/0/default.jpg"
-    this.imageIDs = this.hasItems
-      ? canvases.map((canvas) => {
-          return canvas.id.split("/")[5];
-        })
-      : [];
-
-    const metadata = Array.from(parser.iterateManifestMetadata());
-    const manifestMetadataHash = {};
-
-    if (metadata) {
-      for (const field of metadata) {
-        const label = field.label["en"][0];
-        const value = field.value["en"];
-        manifestMetadataHash[label] = value;
-      }
-    }
 
+    // Non-Manifest/Metadata related fields
     this.uuid = uuid;
     this.link =
       process.env.APP_ENV === "development" || process.env.APP_ENV === "qa"
         ? `https://qa-digitalcollections.nypl.org/items/${this.uuid}`
         : `https://digitalcollections.nypl.org/items/${this.uuid}`;
-    this.typeOfResource = manifestMetadataHash["Resource Type"]
-      ? manifestMetadataHash["Resource Type"].toString()
+    this.manifestURL = `${process.env.COLLECTIONS_API_URL}/manifests/${uuid}`;
+
+    // Manifest related fields
+    this.hasItems = manifest.items.length > 0 ? true : false;
```

Comment:
nit: I know this is copied but could just do `this.hasItems = manifest.items.length > 0`